addons/web/controllers/main.py

```diff
@@ -1420,15 +1420,6 @@ def content_assets(self, id=None, filename=None, unique=None, extra=None, **kw):
         return self._get_content_common(xmlid=None, model='ir.attachment', id=id, field='datas', unique=unique, filename=filename,
             filename_field='name', download=None, mimetype=None, access_token=None, token=None)
 
-    @http.route(['/web/partner_image',
-        '/web/partner_image/<int:rec_id>',
-        '/web/partner_image/<int:rec_id>/<string:field>',
-        '/web/partner_image/<int:rec_id>/<string:field>/<string:model>/'], type='http', auth="public")
-    def content_image_partner(self, rec_id, field='image_128', model='res.partner', **kwargs):
-        # other kwargs are ignored on purpose
-        return self._content_image(id=rec_id, model='res.partner', field=field,
-            placeholder='user_placeholder.jpg')
```

Comment:
`_content_image` and `_content_image_get_response` can probably lose the `placeholder` parameter now? I think this was the only usage.